### PR TITLE
Task/3554/Remove beta access resources

### PR DESCRIPTION
**Pull request by pcoder**
Clean up of Beta access urls, forms, models, classes. Mainly concerns with the Dcl landing and whydcl pages.

### CODE REVIEW

File: datacenterlight/views.py [hunk 1]
```diff
@@ -99,108 +98,8 @@ def get(self, request, *args, **kwargs):
...
-            'name': data.get('name'),
-            'vms': vms
-        }
-
-        email_data = {
-            'subject': 'DatacenterLight Beta Access Request',
-            'from_address': '(datacenterlight) datacenterlight Support <support@datacenterlight.ch>',
-            'to': 'info@ungleich.ch',
-            'context': context,
-            'template_name': 'request_beta_access_notification',
```
Comment (tiwariav) [inline datacenterlight/views.py]: related email template file `datacenterlight/templates/emails/request_beta_access_notification.html` can also be removed.

File: datacenterlight/urls.py [hunk 2]
```diff
@@ -17,15 +17,12 @@
     url(r'^l/$', IndexView.as_view(), name='index_l'),
     url(r'^whydatacenterlight/?$', WhyDataCenterLightView.as_view(),
         name='whydatacenterlight'),
-    url(r'^beta-program/?$', BetaProgramView.as_view(), name='beta'),
     url(r'^landing/?$', LandingProgramView.as_view(), name='landing'),
     url(r'^payment/?$', PaymentOrderView.as_view(), name='payment'),
     url(r'^order-confirmation/?$', OrderConfirmationView.as_view(),
         name='order_confirmation'),
     url(r'^order-success/?$', SuccessView.as_view(), name='order_success'),
```
[the remark this answers is not in the document]
Comment (pcoder) [reply datacenterlight/urls.py]: Which view/template are you referring to here @tiwariav ?

[the remark this answers is not in the document]
Comment (tiwariav) [reply datacenterlight/urls.py]: `SuccessView`
earlier it after a successful order, this view showed a page with a success message. Now we are showing the message in a modal and redirecting to Dashboard.

[the remark this answers is not in the document]
Comment (pcoder) [reply datacenterlight/urls.py]: Hmm, you seem to be right .. Not related to beta access, but as we are not using it, I will clean it up here. Thanks.

**Pull request merged**